packages/sys-apps/cpuid/cpuid-20201006.exheres-0

```diff
@@ -17,12 +17,19 @@ MYOPTIONS="parts: binaries documentation"
 
 DEPENDENCIES=""
 
-BUGS_TO="virkony@gmail.com cpuid@etallen.com"
+src_prepare() {
```

Comment:
Too lazy to write a proper patch (which would also cancel manual src_install).

Reply:
Not sure I understand about effect on manual `src_install`. If it is possible to get rid of explicit `src_install` - I'm fine with that.

Would it make more sense to change make targets instead of patching default target? E.g. 
```sh
DEFAULT_SRC_COMPILE_PARAMS=( {,cpuinfo2}cpuid{,.man} )
````

We anyway specify them explicitly in `src_install` right now.

I don't mind `sed` also, but it is just a bit more brittle (imagine next version using `cpuid.man.xz`). For patch that allows disabling man compression it seems that we'll need to send it over e-mail.